src/js/ia-topnav/ia-topnav.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-undef */
```

Comment:
Ooooh, ia-topnav is too much :P We don't this one.

Reply:
noted